doc/modules/model_evaluation.rst

```diff
@@ -1033,6 +1033,60 @@ set [0,1] has an error: ::
     for an example of zero one loss usage to perform recursive feature
     elimination with cross-validation.
 
+.. _brier_score_loss:
+
+Brier score loss
+----------------
+
+The :func:`brier_score_loss` function computes the
+`Brier score <https://en.wikipedia.org/wiki/Brier_score>`_
+for binary classes. Quoting Wikipedia:
+
+
+    "The Brier score is a proper score function that measures the accuracy of
+    probabilistic predictions. It is applicable to tasks in which predictions
+    must assign probabilities to a set of mutually exclusive discrete outcomes."
+
+This function returns a score of the mean square difference between the actual outcome and
+the predicted probability of the possible outcome. The actual outcome has to be 1 or 0
+(true or false), while the predicted probability of the actual outcome can be
+a value between 0 and 1.
+The brier score loss is also between 0 to 1 and the lower the score (the mean square difference is smaller),
+the more accurate the prediction is. It can be thought of as a measure of the "calibration"
+of a set of probabilistic predictions.
+
+.. math::
+
+   BS = \frac{1}{N} \sum_{t=1}^{N}(f_t - o_t)^2
+
+where : :math:`N` is the total number of predictions, :math:`f_t` is the predicted
+probablity of the actual outcome :math:`o_t`
```

Comment:
It's a detail, but I prefer if, before a code block, you add "::", which is the explicit way of defining a code-block, rather than relying on indentation, which is fragile.